packages/examples/cvat/exchange-oracle/src/db/errors.py

```diff
@@ -3,6 +3,5 @@
 if db_engine.driver != "psycopg2":
     raise NotImplementedError
 
-from psycopg2.errors import LockNotAvailable
```

Comment:
This import is supposed for db driver abstraction. I suggest to add `__all__ = [LockNotAvailable]` instead of removing.